workshop.md

```diff
@@ -91,162 +95,173 @@ Once you are in the new Lakehouse, create a new notebook by selecting `Open Note
 
 Once the notebook is created, select the `Save as` icon and save the notebook as `flashcards_workshop`. 
 
-## Install required libraries
-
-Once the notebook is saved, add the following code to the first cell to install the required libraries:
-
-```bash
-%pip install qrcode
-```
-
-We are going to use the `qrcode` library to generate the QR codes for the flashcards. We do this as a first step, because once the library is installed, the notebook will automatically restart the kernel to make the library available for the rest of the code.
-
----
-
 # Import Data to your Lakehouse
 
-The first step is to import the data from an external source into your Lakehouse. For this workshop, we will use Microsoft Learn modules as our source material. We'll fetch the learn module Markdown files from the Microsoft Learn GitHub repository and import them into our Lakehouse.
+Let's say we want to practice our knowledge on the [Get started with Real-Time Analytics in Microsoft Fabric](https://learn.microsoft.com/training/modules/get-started-kusto-fabric/?WT.mc_id=javascript-76678-cxa) module. The source material for this module is available in the [Microsoft Learn GitHub repository](https://github.com/MicrosoftDocs/learn/). In this case navigate to the module folder [learn-pr/wwl/get-started-kusto-fabric](https://github.com/MicrosoftDocs/learn/tree/main/learn-pr/wwl/get-started-kusto-fabric). There you will find an includes folder that contains the files in the module. This is a list all the modules in their respective Markdown files.
```

Comment:
this is a list _of_ all the _units_ in their respective Markdown files. Learn Modules are made of units.

Reply:
added the correct text